File: source/Finding An A/findingana.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    string s;
    cin >> s;
    bool flag = false;
    for (int i=0; i<s.length(); i++) {
        if (s[i] == 'a') {
            flag = true;
        }

        if (flag) {
            cout << s[i];
        }
    }
    return 0;
}

File: source/Finding An A/findingana.go
package main

import "fmt"

func main() {
	var s string
	fmt.Scanln(&s)
	flag := false
	for i := 0; i < len(s); i++ {
		if s[i] == 'a' {
			flag = true
		}
		if flag {
			fmt.Printf("%c", s[i])
		}

	}
}
